erpnext/stock/doctype/landed_cost_voucher/test_landed_cost_voucher.py

```diff
@@ -250,6 +251,38 @@ def test_multi_currency_lcv(self):
 			self.assertEqual(entry.credit, amounts[0])
 			self.assertEqual(entry.credit_in_account_currency, amounts[1])
 
+	def test_asset_lcv(self):
+		"Check if LCV for an Asset updates the Assets Gross Purchase Amount correctly."
+		if not frappe.db.exists("Asset Category", "Computers"):
+			create_asset_category()
+
+		if not frappe.db.exists("Item", "Macbook Pro"):
+			create_fixed_asset_item()
+
+		pr = make_purchase_receipt(item_code="Macbook Pro", qty=1, rate=50000)
+
+		# check if draft asset was created
+		assets = frappe.db.get_all('Asset', filters={'purchase_receipt': pr.name})
+		self.assertEqual(len(assets), 1)
+
+		frappe.db.set_value("Company", pr.company, "capital_work_in_progress_account", "CWIP Account - _TC")
```

Comment:
```suggestion
		
		frappe.db.set_value("Company", pr.company, "capital_work_in_progress_account", "CWIP Account - _TC")

		pr = make_purchase_receipt(item_code="Macbook Pro", qty=1, rate=50000)

		# check if draft asset was created
		assets = frappe.db.get_all('Asset', filters={'purchase_receipt': pr.name})
		self.assertEqual(len(assets), 1)

```

Move setting of account before creating PR? 

Test becomes flaky depending on which tests ran. Example: https://github.com/frappe/erpnext/runs/3277810413

Reply:
@ankush https://github.com/frappe/erpnext/pull/26873